entity-framework/core/providers/sqlite/limitations.md

```diff
@@ -17,6 +17,25 @@ The common relational library (shared by Entity Framework relational database pr
 * Sequences
 * Computed columns
 
+## Query limitations
+
+SQLite doesn't natively support the following data types. EF Core can read and write values of these types, and querying for equality (`where e.Property == value`) is also support. Other operations, however, like comparison and ordering will require evaluation on the client.
```

Comment:
> EF Core can read and write values of these types, and querying for equality (`where e.Property == value`) is also support.

Reply:
Meh. Gotta leave contributors some improper grammar to correct. 😉